common/persistence/src/main/resources/db/migration/V22_alterTableForCheckInProtectedReports.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE check_in_protected_reports ADD COLUMN mac BYTEA NOT NULL;
```

Comment:
This will be problem. All existing entries will have a null value but the constraint is not null.

Reply:
Also I would rename this migration to reflect what changed about the table ( alterTableCheckInProtectedReportsAddMac or something like that